test/Window/Context.test.cpp

```diff
@@ -1,8 +1,61 @@
 #include <SFML/Window/Context.hpp>
 
+// Other 1st party headers
+#include <SFML/Window/ContextSettings.hpp>
+
+#include <doctest/doctest.h>
+
+#include <WindowUtil.hpp>
+#include <string>
 #include <type_traits>
 
 static_assert(!std::is_copy_constructible_v<sf::Context>);
 static_assert(!std::is_copy_assignable_v<sf::Context>);
 static_assert(!std::is_nothrow_move_constructible_v<sf::Context>);
 static_assert(!std::is_nothrow_move_assignable_v<sf::Context>);
+
+TEST_CASE("[Window] sf::Context" * doctest::skip(skipDisplayTests))
+{
+    SUBCASE("Construction")
+    {
+        const sf::Context context;
+
+        CHECK(context.getSettings().majorVersion > 0);
+    }
+
+    SUBCASE("Version String")
+    {
+        sf::Context context;
+
+        CHECK(context.setActive(true));
+
+#if defined(SFML_SYSTEM_WINDOWS)
+        using glGetStringFuncType = const char*(__stdcall*)(unsigned int);
+#else
+        using glGetStringFuncType = const char* (*)(unsigned int);
+#endif
```

Comment:
```suggestion
#if defined(SFML_SYSTEM_WINDOWS)
#define STDCALL __stdcall
#else
#define STDCALL 
#endif

using glGetStringFuncType = const char*(STDCALL*)(unsigned int);
```

Makes it more clear that both branches are identical except for `__stdcall`.

Reply:
Fixed.